src/pages/Cart/Cart.js

```diff
@@ -16,13 +16,11 @@ class Cart extends Component {
   }
 
   componentDidMount() {
-    const { isEmptyCart } = this.state;
-    if (isEmptyCart.length > 0) {
-      <OrderTable />;
-    } else {
-      <isEmptyCart />;
-    }
-    this.setState({ isEmptyCart: !isEmptyCart });
+    fetch('http://localhost:3000/data/cartListData.json')
```

Comment:
```suggestion
    fetch('/data/cartListData.json')
```
절대경로로 시작하면 자동으로 현재 우리 앱이 돌아가고 있는 주소로 인식이 됩니다! 지금처럼 명시적으로 3000포트까지 다 적어준 경우에는 만약 3000포트에 다른 웹을 실행하고 있어서 3001포트에서 실행하게 되는 등 다른 주소에서 실행하게 되면 에러가 발생할 수 있습니다

Reply:
넵 확인 감사드립니다. 다음에는 조금 더 PR에 구현사항을 길게 쓰겠습니다.
목데이타 돌릴 때는 해당주소까지 포함해서 하는 걸로 알고 있었습니다.